Sources/Sentry/include/SentryScope+Private.h

```diff
@@ -20,7 +20,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (atomic, strong) SentryUser *_Nullable userObject;
 
-@property (atomic, strong) SentryPropagationContext *propagationContext;
+@property (nonatomic, strong) SentryPropagationContext *propagationContext;
```

Comment:
`m`: Why did you change this? If there is a specific reason, it would be great if you could add comment so we know in the future